.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: false
+language: node_js
+node_js:
+  - "5.0"
+
```

Comment:
Are we going to start testing with Travis for this? I think we used to use Travis to run tests for the previous JS SDK.

Reply:
We can.  I think they also have a Jenkins plugin that may be a good fit for us when we get the new and improved Jenkins.  I sent it to Ben yesterday, and he said it looked reasonable.

Reply:
Curious to hear what the benefit would be to running these tests in travis? Makes more sense to me in terms of consistency and simplicity to run these tests as part of the existing dashboard build job, nodejs is the only runtime requirement(as all the testing is executed on a remote service) - which I think Jenkins is already setup with for being able to build our other nodejs projects or we can execute these tests in a docker container as a fallback

Reply:
The benefit as I understand it now is that we do not have a deployment process for the JS SDK beyond a standard npm publish. This step is typically done from a developer machine, not through a build tool. Also, since the repo is public, other contributions can come in from outside developers. To that end I like having something like Travis run on every PR that comes, where our current Jenkins set up can not do this. I'm fine adjusting this workflow if we want to bring Jenkins into the mix, but it isn't right now and requires manual trigger the way it is working at the moment.

Reply:
👍 Makes sense, travis seems like a better and more natural fit for this sort of non Tozny service workflow